### DISCUSSION TITLE
ETA for a version of gulp and glob-watcher compatible with Node 14

### DISCUSSION THREAD

**Discussion by stof** [gulpjs/gulp#2506] [General] [answered]
Node 14 will become the active node LTS next week (and is the active stable version since 6 months). But currently, gulp is incompatible with it due to `glob-watcher` relying on `chokidar` 2.x rather than 3.x.
There are existing issues about that (https://github.com/gulpjs/glob-watcher/issues/55 and others linked in it) but they are closed for comments.

Is there any ETA for a release of gulp upgrading chokidar (and dropping support for old node versions due to that) ? To me, supporting the active node LTS seems more important than supporting node 0.10. A project incompatible with the maintained versions of node seems like a project that will die, as I don't see companies staying on unmaintained versions of node to be able to keep using it.

**Comment by phated** [comment, marked as answer]
There is no news. As with most open source projects, this is supported by people that work in the spare time. We have a plan amongst the team for gulp 5 that includes what you asked about, but none of the core team has had time to work on their parts. None of the community has stepped up to help with things either.

An aside: due to the pandemic, we've almost lost all of our financial backers, too, so there isn't much incentive for people to work on things.

**Reply by stof** [reply]
> We have a plan amongst the team for gulp 5 that includes what you asked about, but none of the core team has had time to work on their parts. None of the community has stepped up to help with things either.

is this plan public anywhere ?

**Reply by phated** [reply]
All our projects are tracked here on our github org: https://github.com/orgs/gulpjs/projects

Specifically the v5 stuff is at https://github.com/orgs/gulpjs/projects/4 and we'd like to pull in the [v4 stuff](https://github.com/orgs/gulpjs/projects/1) where possible.

**Reply by stof** [reply]
what about a v5 focused mainly on dropping support for old nodejs version and supporting node 14 instead, if there is not enough resources to do other cleanups in gulp ?

**Reply by phated** [reply]
No. You are welcome to contribute.
